src/scripts/destinyTrackerApi/reviewSubmitter.js

```diff
@@ -0,0 +1,70 @@
+import { itemTransformer } from './itemTransformer.js';
+
+class reviewSubmitter {
```

Comment:
I think it'd be nice to just have a single service object representing the DTR rating/score API rather than having a bunch of independent objects. See `dimBungieService` for an example.